Package.swift

```diff
@@ -37,6 +37,5 @@ var swiftSettings: [SwiftSetting] {
     [
         .enableUpcomingFeature("ExistentialAny"),
         .enableUpcomingFeature("MemberImportVisibility"),
```

Comment:
Might as well add the full set of upcoming flags:
```suggestion
        .enableUpcomingFeature("MemberImportVisibility"),
        .enableUpcomingFeature("InternalImportsByDefault"),
        .enableUpcomingFeature("InferIsolatedConformances"),
        .enableUpcomingFeature("NonisolatedNonsendingByDefault"),
        .enableUpcomingFeature("ImmutableWeakCaptures"),
```
Most of those won't have any effect on this package anyway, but doesn't hurt to have 'em.

(Apologies in advance if any of these turn out to be invalid in 6.1. I forget what SwiftPM's rules are for that.)